src/Common/src/Interop/Unix/libssl/StreamSizes.cs

```diff
@@ -5,15 +5,32 @@ namespace System.Net
 {
     internal class StreamSizes
     {
+        private static readonly int s_header;
+        private static readonly int s_trailer;
+        private static readonly int s_maximumMessage;
+
         public int header;
         public int trailer;
         public int maximumMessage;
 
-        internal StreamSizes(int headerSize, int trailerSize, int maxMessageSize)
+        static StreamSizes()
+        {
+            // TODO (Issue #3362) : Trailer size requirement is changing based on protocol
+            //       SSL3/TLS1.0 - 68, TLS1.1 - 37 and TLS1.2 - 24
+            //       Current usage is only to compute max input buffer size for
+            //       encryption and so the native code returns the max
```

Comment:
I put it both here in managed code and in the native code. Officially it only belongs with the native code, but I also copied it here in case whoever is looking for TODO Issue #3362 comments only search in *.cs or in the VS solution. This way it still gets picked up.